backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -155,6 +155,11 @@ def spree_dom_id(record)
         dom_id(record, 'spree')
       end
 
+      def admin_layout(layout = nil)
+        @admin_layout = layout if layout
+        @admin_layout
+      end
+
```

Comment:
As I look at this code more though, the name `admin_layout` is a bit confusing to me -- I thought it would be a Rails layout template, but it looks like it's actually supposed to be the name of a _class_, yes?